setup.py

```diff
@@ -95,7 +93,11 @@
 
 dev_all = install_requires + dev_requires + docs_requires
 
-yatai_service = install_requires + aws_sam_cli + postgres + azure_cli
+yatai_service = (
+    ["google-cloud-storage", "azure-cli", "psycopg2", "psycopg2-binary"]
+    + install_requires
+    + aws_sam_cli
```

Comment:
@yubozhao  is aws_sam_cli still used in tests?  can we get rid of it in unit tests?